### PR TITLE
Update specfile for 1.4 release

### PR DESCRIPTION
**Pull request by praveenkumar**
Update spec file for new release.

### CODE REVIEW

File: adb-utils.spec [hunk 1]
```diff
@@ -44,6 +44,9 @@ ln -s /opt/adb/add_insecure_registry %{buildroot}%{_bindir}/add_insecure_registr
 %doc LICENSE  README.rst
 
 %changelog
+* Fri Apr 01 2016 kumarpraveen.nitdgp@gmail.com 1.4-1
```
Comment (navidshaikh) [inline adb-utils.spec]: I would suggest you add your name as well along with email id.

File: adb-utils.spec [hunk 2]
```diff
@@ -44,6 +44,9 @@ ln -s /opt/adb/add_insecure_registry %{buildroot}%{_bindir}/add_insecure_registr
...
+- Update to 1.4 with couple of bugfix
```
Comment (navidshaikh) [inline adb-utils.spec:48]: IMO, you should list down the bugs fixed.

Comment (praveenkumar) [inline adb-utils.spec:48]: Specfile should not list bug fixes for a release, if there is minor version bump then we list specific bugs which caused it.

File: adb-utils.spec (hunk 1 above)
Comment (praveenkumar) [inline adb-utils.spec]: Done.

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**